R/dataDrivenClusters.R

```diff
@@ -27,35 +27,28 @@ dataDrivenClusters <- function(voxel_df, n_pca = 20, n_umap = 2, n_clust = 2,
   umap_df <- as.data.frame(umap_coords)
 
   #apply kmeans to umap-reduced data
```

Comment:
can you put a space after # in your comments?

Reply:
@galbby5

Reply:
yeah that is a bad habit of mine